Make black remove leading and trailing spaces from one-line docstrings: fixes #1738 #1812 (#1740)
psf/black: CHANGES.md

#### _Black_

> - `Black` now requires one-line docstrings to be stripped (PR #1740)

Reviewer: ```suggestion
- `Black` now strips leading and trailing spaces from one-line docstrings (PR #1740)
```
Black doesn't require anything, it does it all for you
